protected/humhub/modules/admin/models/forms/AuthenticationSettingsForm.php

```diff
@@ -96,6 +96,16 @@ public function attributeLabels()
         ];
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function attributeHints()
+    {
+        return [
+            'internalRequireApprovalAfterRegistration' => Yii::t('AdminModule.user', 'If enabled, you can select the managers in the group settings.'),
```

Comment:
From: `If enabled, you can select the managers in the group settings.`
To: `If enabled, you can select group managers in the group settings`